src/Types.hs

```diff
@@ -14,19 +14,24 @@ module Types where
 
 import           Control.Applicative
 import           Data.Aeson
-import           Data.Map            as M
+import qualified Data.ByteString      as BS
+import           Data.ByteString.Lazy (ByteString, fromStrict)
+import           Data.Map             as M
 import           Data.Maybe
-import           Data.Text           as T
+import           Data.Text            as T
 import           Data.Time
-import           Hasql.Pool          as P
+import           Data.Typeable
+import           Hasql.Pool           as P
+import qualified Network.HTTP.Media   as M
```

Comment:
This might just my OCD/pedantry - but it's probably clearer if we don't alias multiple modules using the same letter (see `Data.Map` above).